app/server/api/auth_api.py

```diff
@@ -768,12 +769,31 @@ def post(self):
         if not organisation:
             response_object = {'message': 'Organisation Not Found'}
             return make_response(jsonify(response_object)), 404
-        email_exists = EmailWhitelist.query.filter(func.lower(EmailWhitelist.email)==email).first()
 
-        if email_exists:
-            response_object = {'message': 'Email already on whitelist.'}
+        email_exists_for_org = EmailWhitelist.query.filter(func.lower(EmailWhitelist.email)==email).first()
+        if email_exists_for_org:
+            response_object = {'message': 'Email already on organisation whitelist.'}
             return make_response(jsonify(response_object)), 400
 
+        email_exists = EmailWhitelist.query.filter(func.lower(EmailWhitelist.email)==email)\
+            .execution_options(show_all=True).first()
+        if email_exists and not email_exists.used:
```

Comment:
what bad thing happens if we remove this check and a user can be added to a whitelist for more than one org? Presumably it breaks the signup if the user tries to use both invites, but we could fix that. feels better than having one org depend on another org's behaviour (can't resend invites for the user etc etc). Also means it doesn't allow admins discover who's been added to another org.

Reply:
it won't sign up. the user will be associated to one account. i didn't want to do it as could be complex (need to rewrite part of the auth flow) and burn a day or so on implementing, whereas this was a few hours.

EmailWhitelist is currently OneOrgBase.

Reply:
hmmm i might take a quick squiz at the auth flow and see how complex it'd be. I think the expected behavor from a ux PoV is that an invite shouldn't be blocked by the presence or lack of one on another org, unless you feel otherwise?

Reply:
definitely agree it's currently non-ideal UX - just seems pretty edge case at present. that would be great :)

Reply:
my opinion is weakened by the fact there's a bunch of other dependencies (tiers, deleting etc), but ideally we'd fix those too